FinSearchUnified/Tests/BusinessLogic/Models/FindologicArticleModelTest.php

```diff
@@ -1649,4 +1650,254 @@ public function testImageUrlsWithSpecialCharactersAreEncoded()
             }, $images)
         );
     }
+
+    public function mainVariantExportOrderProvider()
+    {
+        $mainVariantOnTheBeginArticleConf = [
```

Comment:
Please create one base config and define the other ones with `array_merge($baseConfig, ...)`